tests/aws/aws_databases.rs

```diff
@@ -522,7 +522,7 @@ fn deploy_a_working_environment_with_mongodb_v4_4() {
 /// test mongodb v3.6 with production environment (DocumentDB)
 #[test]
 #[ignore]
-fn deploy_a_working_production_environment_mongodb_v3_6() {
+fn deploy_a_working_environment_with_production_mongodb_v3_6() {
```

Comment:
we should define a right syntax for this I saw:

mysql_deploy_a_working_environment,
mongodb_v4_4_deploy_a_working_environment,
deploy_a_working_environment_with_production_mongodb_v3_6

I suggest:
[staging/production]_env_[database_type]_[version?] -> with optional version fields

Reply:
You're right